core/domain/build.gradle.kts

```diff
@@ -32,6 +32,7 @@ dependencies {
     implementation(projects.core.rule)
 
     testImplementation(projects.core.testing)
+    implementation(libs.androidx.lifecycle.livedata.ktx)
```

Comment:
I don't know the reason for importing a new dependency here

Reply:
For using asFlow()

Reply:
Is itnecessary to do it?